daft/dataframe/dataframe.py

```diff
@@ -286,7 +286,9 @@ def iter_rows(self, results_buffer_size: Optional[int] = NUM_CPUS) -> Iterator[D
                     yield row
 
     @DataframePublicAPI
-    def to_arrow_iter(self, results_buffer_size: Optional[int] = 1) -> Iterator["pyarrow.RecordBatch"]:
+    def to_arrow_iter(
+        self, results_buffer_size: Optional[int] = 1, cast_tensors_to_ray_tensor_dtype: bool = False
```

Comment:
should we document these arguments?

Reply:
Yeah good point... I kind of want to remove them at some point though they're not really useful and add a bunch of tech debt

Reply:
Actually for this case I'll make them "private" then by adding an underscore

Reply:
(Removed in #2802 )